.github/workflows/docs.yml

```diff
@@ -11,7 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/setup-python@v5
+      with:
+        python-version: 3.11
```

Comment:
Without this, it'll use Python 3.12. `versioneer.py` file needs to be updated for that to work with 3.12.

Reply:
@skshetry Dropped versioneer, please rebase and you will be able to safely remove this.